integration-tests/src/test/burn.test.ts

```diff
@@ -0,0 +1,117 @@
+import { Blockchain, KeyringPair, creditcoinApi, BN } from 'creditcoin-js';
+import { forElapsedBlocks, testData } from 'creditcoin-js/lib/testUtils';
+import { CreditcoinApi } from 'creditcoin-js/lib/types';
+import { mnemonicGenerate } from '@polkadot/util-crypto';
+import { extractFee, describeIf } from '../utils';
+
+describeIf((global as any).CREDITCOIN_EXECUTE_SETUP_AUTHORITY, 'burn', () => {
+    let ccApi: CreditcoinApi;
+    let sudoSigner: KeyringPair;
+    let wallet: KeyringPair;
+
+    const testingData = testData(
+        (global as any).CREDITCOIN_ETHEREUM_CHAIN as Blockchain,
+        (global as any).CREDITCOIN_CREATE_WALLET,
+    );
+    const { keyring } = testingData;
+
+    beforeAll(async () => {
+        ccApi = await creditcoinApi((global as any).CREDITCOIN_API_URL);
+        sudoSigner = (global as any).CREDITCOIN_CREATE_SIGNER(keyring, 'lender');
```

Comment:
this function also accepts `'sudo'` as argument, see `globalSetup.ts`. 

Let's use that for better readability!

I have send a separate PR to address all other files inside the test suite, see https://github.com/gluwa/creditcoin/pull/1436.